pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/ArrayInitializationVerbosenessRule.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.codestyle;
+
+import net.sourceforge.pmd.lang.java.ast.ASTLocalVariableDeclaration;
+import net.sourceforge.pmd.lang.java.rule.AbstractJavaRule;
+
+public class ArrayInitializationVerbosenessRule extends AbstractJavaRule {
+
+    @Override
+    public Object visit(ASTLocalVariableDeclaration node, Object data) {
+
+        if (node.isArray() && node.getImage() != null) {
+            if (node.getImage().matches("(= ?new).+\\{")) {
```

Comment:
the image will actually always be null. Image checking is a poor way of dealing with code structure and should be avoided. The proper way would be to use the `visit(ASTVariableInitializer, Object)` method, and check if the initializer node has a  `ASTAllocationExpression` descendant